src/main/java/io/jenkins/plugins/analysis/core/steps/IssuesRecorder.java

```diff
@@ -477,7 +500,7 @@ public boolean perform(final AbstractBuild<?, ?> build, final Launcher launcher,
      * that has Pipeline-specific behavior.
      */
     void perform(@NonNull final Run<?, ?> run, @NonNull final FilePath workspace,
-            @NonNull final TaskListener listener, @NonNull final StageResultHandler statusHandler)
+                 @NonNull final TaskListener listener, @NonNull final StageResultHandler statusHandler)
```

Comment:
is this formatting correct? Why did you change this line, it seems unrelated?

Reply:
Removed :+1: